migrations/20190409210930_user_table.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable('user', (tbl)=>{
+    tbl.increments();//primary key command defaults as ID
+
+    tbl.string('user',128)
+       .notNullable()
+       .unique();
+
+    tbl.string('password',128)
+       .notNullable()
+       .unique();
+
+  
+
+   
+  
```

Comment:
There's a lot of unnecessary white space here! I recommend using an extension like Prettier to keep your code formatting consistent.